WeakAuras/Prototypes.lua

```diff
@@ -1488,7 +1577,12 @@ WeakAuras.event_prototypes = {
             end
             state.changed = true;
           elseif (event == "UNIT_DISPLAYPOWER") then
-            local spellID = select(9, UnitCastingInfo("player"));
+            local spellID;
+            if WeakAuras.IsClassic() then
+              spellID = select(9, CastingInfo());
+            else
+              spellID = select(9, UnitCastingInfo("player"));
+            end
             if spellID then
               local costTable = GetSpellPowerCost(spellID);
```

Comment:
Strange that GetSpellPowerCost is available on classic.

You can use WeakAuras.UnitCastingInfo("player") above though.

Reply:
it exists https://i.imgur.com/aV4BxG6.png